### ISSUE TITLE
MacTeX 2019 TeX Live Utility Version Conflict

### ISSUE THREAD

**Issue by marcschwartz** [amaxwell/tlutility#73]
Hi,

Just updated to TeXLive 2019, after removing 2018.

The install appears to be fine, however, I get the error message in the attached screen capture when attempting to run the TLU, which is version 1.35, and I get a Listing Failed error message.

I am running on a MBP, with macOS 10.14.5, just updated yesterday.

From the CLI, I get an indication that I am running TeXLive 2019:

~  pdflatex
This is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019) (preloaded format=pdflatex)
 restricted \write18 enabled.
**

and the downloaded PKG file passed the hash check.

If I click on Upgrade, it takes me to the MacTeX site.

If I click on Ignore, I get the Listing Failed error.

If I click on Check for Updates in the menu, I get an indication that 1.35 is the newest version of Tex Live.

I am currently using  http://mirror.ctan.org/systems/texlive/tlnet for the repo.

Any thoughts on this? I have been running MacTeX for a number of years and this is the first time that I have had this issue after an upgrade.

Thanks,

Marc

<img width="424" alt="Screen Shot 2019-05-14 at 7 37 45 AM" src="https://user-images.githubusercontent.com/1749084/57709725-8c442380-7639-11e9-8691-d7cb8c8e1352.png">

**Comment by amaxwell** [comment]
If you're using MacTeX, make sure Configure -> Change Default TeX Live Version… is set to TL 2019, and make sure the path in TLU's preferences points to `/Library/TeX/texbin` and not somewhere in the bowels of `/usr/local`.

**Comment by marcschwartz** [comment]
Hi Adam,

Thanks for the quick reply.

2019 was the only version indicated in the configuration:

![Screen Shot 2019-05-14 at 12 16 02 PM](https://user-images.githubusercontent.com/1749084/57714651-f614fb00-7642-11e9-9970-1ebac4dff0d8.png)

However, your second clue was the issue. I have adjusted that now and all appears to be working fine. 

Not sure how and/or when that got changed, but as noted, had not been an issue in all these years.

Thanks again!

Marc
